src/components/app.js

```diff
@@ -45,24 +45,18 @@ export const SmallAppDetails = ({
     <React.Fragment>
       <Typography component="span" variant="body2" color="textPrimary">
         {description}
+      </Typography>
+      <Typography component="span" variant="body2">
         {!hideRewards && (
           <>
             <br />
-            rewards: {earnings}{' '}
-          </>
-        )}
-        {showSourceLink && openSourceUrl && (
-          <>
-            <br />
-            <a href={openSourceUrl} style={styles.link}>
-              Source code
-            </a>
+            Rewards: <b>{earnings}</b>{' '}
```

Comment:
The source code link got lost.

Reply:
we don't need it here! I removed it for a reason!

Reply:
What is the reason?

Reply:
this is too much info and adds a lot of UI issues, and also if they want to get the open source code they can visit the app details where they can try the app and or visit the source code like product hunt

Reply:
It is used in appco-comment and appcodetails.